doc/tutorial/climada_engine_Impact.ipynb

```diff
@@ -66,11 +66,12 @@
     "| Attributes from input | Data Type | Description|\n",
     "| :- | :- | :- |\n",
     "| tag |(dict)| dictionary storing the tags of the inputs (Exposure.tag, ImpactFuncSet.tag Hazard.tag)|\n",
-    "| even_id |list(int)| id (>0) of each hazard event (Hazard.event_id) |\n",
+    "| event_id |list(int)| id (>0) of each hazard event (Hazard.event_id)|\n",
     "| event_name |(list(str))| name of each event (Hazard.event_name)|\n",
     "| date |np.array| date of events (Hazard.date)|\n",
     "| coord_exp |np.array| exposures coordinates [lat, lon] (in degrees) (Exposure.gdf.latidues, Exposure.gdf.longitude)|\n",
-    "| frequency |np.array| annual frequency of events (Hazard.frequency)|\n",
+    "| frequency |np.array| frequency of events (Hazard.frequency)|\n",
+    "| frequency_unit |str| unit of event frequency, by default '1/year', i.e., annual, this attribute is purely informative and does not effect calculations (Hazard.frequency_unit)|\n",
```

Comment:
Is it really needed to the "this attribute is purely informative..." ? For instance, this is also the case for the `event_id`, the `unit` or the `date`. And this can change in the future, in particular if time is integrated more deeply into CLIMADA.

Reply:
Just wanted to emphasize that the frequency_unit is not being considered at any point in calculations. All it really does is change labels in plots. 
As a naive user, when I can set a frequency_unit in an object and I get a frequency, I might naturally suspect the two to be somehow correlated. Which is not the case. Therefore the warning.

Reply:
But this is also the case for other entries. Thus, I would remove it as it is confusing why this is said for that entry, but not for others. Furthermore, there will likely be methods in the future that will actually care about the frequency unit.